### ISSUE TITLE
Automatic Shotgun

### ISSUE THREAD

**Issue by TeslaCoil9001** [studio-minus/ppg-bugs-and-requests#2023]
**Feature description**
An automatic version of the shotgun, probably an AA12.

**Feature purpose**
To have a vanilla version of the shotgun in full auto that still fits the style of the game.

**Additional context**
![image](https://user-images.githubusercontent.com/112362312/187296713-0f5e1d9c-8d8c-49e1-9a7b-b71e15e1c184.png)

**Comment by ATPGaming** [comment]
Duplicate of
https://github.com/studio-minus/ppg-bugs-and-requests/issues/1648

**Comment by DindinYT37** [comment]
Duplicate of or Related to:
* #429
* #1648

**Comment by TeslaCoil9001** [comment]
> Duplicate of or Related to:
> 
> * [SPAS 12 #429](https://github.com/studio-minus/ppg-bugs-and-requests/issues/429)
> * ["Automatic Shotgun" #1648](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1648)



> Duplicate of #1648

It took 2 months to find those duplicates?

**Comment by DindinYT37** [comment]
> > Duplicate of or Related to:
> > 
> > * [SPAS 12 #429](https://github.com/studio-minus/ppg-bugs-and-requests/issues/429)
> > * ["Automatic Shotgun" #1648](https://github.com/studio-minus/ppg-bugs-and-requests/issues/1648)
> 
> > Duplicate of #1648
> 
> It took 2 months to find those duplicates?

No, I just ignored the issue when you originally posted it because it wasn't worth the time. I just did it now because I got a notification and remembered this issue.
